endpoints/get_address_transactions.py

```diff
@@ -60,3 +63,42 @@ async def get_transactions_for_address(
     return {
         "transactions": tx_list
     }
+
+@app.get("/addresses/{kaspaAddress}/transactions/v2",
```

Comment:
should we create a version at the beginning ?

Reply:
You mean `/v2/addresses/{kaspaAddress}/transactions`?

I can also simply change the endpoint to `/addresses/{kaspaAddress}/transactions-full` to avoid versioning

Reply:
I used `/addresses/{kaspaAddress}/full-transactions` at 27235208739ff1f1cb6ec9537a57fce8419e6d0e